EnglishFilter/sections/adservers.txt

```diff
@@ -11,6 +11,7 @@
 ||armlyemoan.com^
 ||midpopedge.com^
 ||yqorexok.xyz^
+||peppy2lon1g1stalk.com^
```

Comment:
$third-party

Reply:
and dbmdoldiqf.com

Reply:
also saqthneauq.com 
Site can be added to this rule: https://github.com/AdguardTeam/AdguardFilters/blob/972f232195c4237e72e9375e28983faf172d2054/EnglishFilter%2Fsections%2Fspecific.txt#L249-L251